x/concentrated-liquidity/incentives.go

```diff
@@ -415,3 +415,86 @@ func (k Keeper) initOrUpdatePositionUptime(ctx sdk.Context, poolId uint64, posit
 
 	return nil
 }
+
+// collectIncentives collects incentives for all uptime accumulators for all positions belonging to `owner` in the given range.
```

Comment:
When I first read this description, I thought that this func looped over the entire range of ticks and returned every position that owner had. For example, if I set the lower and upper ticks to min and max respectively, it would return every position that user had, not just the ones that were explicitly set on the min and max tick.

Reply:
Hmm I'll move up "in the given range" to be earlier in the description. Let me know if you have a better idea!

Reply:
I think the range comment is what is confusing, because technically a -50/50 position is "in the range" of a position that exists on the min and max ticks, but that is not what is getting returned. I think you just need to specify it only returns positions that match the exact range rather than being within the range.

Reply:
I see, thanks for the clarification – would specifying "positions with exactly the same lower and upper ticks" be a better alternative?